configs/SKYSAKURAH7/config.h

```diff
@@ -0,0 +1,216 @@
+#define MOTOR5_PIN           PA0
+#define MOTOR6_PIN           PA1
+#define MOTOR7_PIN           PE5
+#define MOTOR8_PIN           PE6
+
+//additional pwm out
+#define SERVO1_PIN           PD12
+#define SERVO2_PIN           PD13
+#define SERVO3_PIN           PD14
+#define SERVO4_PIN           PD15
+
+//sbus input
+#define RX_PPM_PIN           PB8
+
+//external led 
+#define LED_STRIP_PIN        PA8
+
+//uart
+#define UART1_TX_PIN         PA9
+#define UART1_RX_PIN         PA10
+#define UART2_TX_PIN         PD5
+#define UART2_RX_PIN         PD6
+#define UART3_TX_PIN         PD8
+#define UART3_RX_PIN         PD9
+#define UART4_TX_PIN         PB9
+#define UART4_RX_PIN         PB8
+#define UART6_TX_PIN         PC6
+#define UART6_RX_PIN         PC7
+#define UART7_TX_PIN         PE8
+#define UART7_RX_PIN         PE7
+#define UART8_TX_PIN         PE1
+#define UART8_RX_PIN         PE0
+
+//i2c
+#define I2C1_SCL_PIN         PB6
+#define I2C1_SDA_PIN         PB7
+#define I2C2_SCL_PIN         PB10
+#define I2C2_SDA_PIN         PB11
+
+//onboard led
+#define LED0_PIN             PB3
+#define LED1_PIN             PB4
+#define LED2_PIN             PB5
+
+//spi1 for mpu6000(deleting)
+#define SPI1_SCK_PIN         PA5
+#define SPI1_SDI_PIN         PA6
+#define SPI1_SDO_PIN         PA7
+
+//external spi
+#define SPI2_SCK_PIN         PB13
+#define SPI2_SDI_PIN         PB14
+#define SPI2_SDO_PIN         PB15
+
+//spi4 for icm42688
+#define SPI4_SCK_PIN         PE12
+#define SPI4_SDI_PIN         PE13
+#define SPI4_SDO_PIN         PE14
+
+//adc
+#define ADC_VBAT_PIN         PC3
+#define ADC_CURR_PIN         PC2
+#define ADC_EXTERNAL1_PIN    PC0
+#define ADC_RSSI_PIN         PC1
+
+//sd card
+#define SDIO_CK_PIN          PC12
+#define SDIO_CMD_PIN         PD2
+#define SDIO_D0_PIN          PC8
+#define SDIO_D1_PIN          PC9
+#define SDIO_D2_PIN          PC10
+#define SDIO_D3_PIN          PC11
+
+//pinio
+#define PINIO1_PIN           PE15
+#define PINIO2_PIN           PD10
+#define PINIO3_PIN           PD11
+
+//sensor CS and EXTI
+#define GYRO_1_EXTI_PIN      PC4
+#define GYRO_2_EXTI_PIN      PB2
+#define GYRO_1_CS_PIN        PA4
+#define GYRO_2_CS_PIN        PE11
+#define BARO_CS_PIN          PC5
+
+/* CS1 pads for SPI2 connection:
+ *
+ * CS1 PB12
+ */
+
+#define TIMER_PIN_MAPPING \
+    TIMER_PIN_MAP( 0, PB0 , 2,  0) \
+    TIMER_PIN_MAP( 1, PB1 , 2,  1) \
+    TIMER_PIN_MAP( 2, PA0 , 2,  2) \
+    TIMER_PIN_MAP( 3, PA1 , 2,  3) \
+    TIMER_PIN_MAP( 4, PA2 , 2,  4) \
+    TIMER_PIN_MAP( 5, PA3 , 2,  5) \
+    TIMER_PIN_MAP( 6, PD12, 1,  6) \
+    TIMER_PIN_MAP( 7, PD13, 1,  7) \
+    TIMER_PIN_MAP( 8, PD14, 1, 12) \
+    TIMER_PIN_MAP( 9, PD15, 1, -1) \
+    TIMER_PIN_MAP(10, PE5 , 1,  0) \
+    TIMER_PIN_MAP(11, PE6 , 1, -1) \
+    TIMER_PIN_MAP(12, PA8 , 1, 14) \
+    TIMER_PIN_MAP(13, PA15, 1,  0) \
+    TIMER_PIN_MAP(14, PC7 , 2,  0) \
+    TIMER_PIN_MAP(15, PC6 , 2,  0) \
+    TIMER_PIN_MAP(16, PB8 , 1,  0) \
+    TIMER_PIN_MAP(17, PB9 , 1,  0)
```

Comment:
This order makes no sense. Servos do not need DMA. Several pins are assigned timers/DMA that are not in use or do not use them. 
```suggestion
#define TIMER_PIN_MAPPING \
    TIMER_PIN_MAP( 0, PA2 , 2,  0) \
    TIMER_PIN_MAP( 1, PA3 , 2,  0) \
    TIMER_PIN_MAP( 2, PB0 , 2,  0) \
    TIMER_PIN_MAP( 3, PB1 , 2,  0) \
    TIMER_PIN_MAP( 4, PA0 , 2,  0) \
    TIMER_PIN_MAP( 5, PA1 , 2,  0) \
    TIMER_PIN_MAP( 6, PE5 , 1,  0) \
    TIMER_PIN_MAP( 7, PE6 , 1,  0) \
    TIMER_PIN_MAP( 8, PD12, 1, -1) \
    TIMER_PIN_MAP( 9, PD13, 1, -1) \
    TIMER_PIN_MAP(10, PD14 , 1,  -1) \
    TIMER_PIN_MAP(11, PD15, 1, -1) \
    TIMER_PIN_MAP(12, PB8 , 1, -1) \
    TIMER_PIN_MAP(13, PA8, 1,  0) 
```